### PR TITLE
Delete tns_modules folder.

### PR DESCRIPTION
**Pull request by e2l3n**
Delete tns_modules folder. 
Add tns-core-modules to package.json.
See issue https://github.com/NativeScript/nativescript-cli/issues/390
And also PR https://github.com/NativeScript/nativescript-cli/pull/805

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,9 @@
 		"type": "git",
 		"url": "git://github.com/NativeScript/template-hello-world.git"
 	},
+	"tns-core-modules": {
```
Comment (Fatme) [inline package.json]: This shouldn't  be here.

Comment (e2l3n) [inline package.json]: Reverted to previous version of the json.

**Comment by Fatme** [comment]
:+1:

**Comment by dtopuzov** [comment]
:+1:

**Pull request merged**